### PR TITLE
6 add availability

### PR DESCRIPTION
**Pull request by hanfak**
connects to #6

### CODE REVIEW

File: app/controllers/spaces.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ class MakersBnb < Sinatra::Base
 
   post '/spaces' do
     user = @current_user ||= User.get(session['user_id'])
-    space = Space.new(name: params[:name], desc: params[:desc], price: params[:price])
+    space = Space.new(name: params[:name], desc: params[:desc], price: params[:price], date: params[:date])
```
Comment (acookson91) [inline app/controllers/spaces.rb]: Can this be changed  to - available_date: params[:available_date]

File: app/models/space.rb [hunk 2]
```diff
@@ -8,6 +8,7 @@ class Space
   property :name, Text
   property :desc, Text
   property :price, Text
+  property :date, String
```
Comment (acookson91) [inline app/models/space.rb]: Can this be changed to - property :date, Time

File: app/views/spaces/new.erb [hunk 3]
```diff
@@ -2,5 +2,6 @@
   <input type="text" name="name" placeholder="Enter space name">
   <input type="text" name="desc" placeholder="Describe your space">
   <input type="text" name="price"placeholder="£ per day">
+  <input type="text" name="date"placeholder="DD/MM/YY">
```
Comment (acookson91) [inline app/views/spaces/new.erb]: Can this be changed to - name="available_date"

**Comment by hanfak** [comment]
:shipit:

**Pull request merged**